nginx/Dockerfile-alpine

```diff
@@ -9,40 +87,132 @@ RUN set -eux; \
+    PORT=80 \
+    NGINX_ALWAYS_TLS_REDIRECT=off \
+    SET_REAL_IP_FROM="127.0.0.1" \
+    REAL_IP_HEADER="X-REAL-IP" \
+    REAL_IP_RECURSIVE="on" \
+    PROXY_TIMEOUT=60s \
+    PROXY_SSL_CERT=/etc/nginx/conf/server.crt \
+    PROXY_SSL_CERT_KEY=/etc/nginx/conf/server.key \
+    PROXY_SSL_DH_BITS=2048 \
+    PROXY_SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
+    PROXY_SSL_CIPHERS="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384" \
+    PROXY_SSL_PREFER_CIPHERS=off \
+    PROXY_SSL_VERIFY=off \
+    PROXY_SSL_OCSP_STAPLING=off \
+    SERVER_NAME=localhost \
+    SSL_PORT=443 \
+    TIMEOUT=60s \
+    WORKER_CONNECTIONS=1024 \
+    LD_LIBRARY_PATH=/lib:/usr/lib:/usr/local/lib \
+    NGINX_ENVSUBST_OUTPUT_DIR=/etc/nginx \
+    # overridden variables
+    USER=nginx \
+    # CRS specific variables
     PARANOIA=1 \
     ANOMALY_INBOUND=5 \
     ANOMALY_OUTBOUND=4 \
     BLOCKING_PARANOIA=1
 
+COPY --from=build /usr/local/modsecurity/lib/libmodsecurity.so.${MODSEC_VERSION} /usr/local/modsecurity/lib/
+COPY --from=build /etc/nginx/modules/ngx_http_modsecurity_module.so /etc/nginx/modules/ngx_http_modsecurity_module.so
+COPY --from=build /usr/share/TLS/server.key /etc/nginx/conf/server.key
+COPY --from=build /usr/share/TLS/server.crt /etc/nginx/conf/server.crt
+COPY --from=build /usr/share/TLS/dhparam-* /etc/ssl/certs/
+COPY --from=build /etc/modsecurity.d/unicode.mapping /etc/modsecurity.d/unicode.mapping
+COPY --from=build /etc/modsecurity.d/modsecurity.conf /etc/modsecurity.d/modsecurity.conf
+COPY --from=crs_release /opt/owasp-crs /opt/owasp-crs
 # We use the templating mechanism from the nginx image here,
 # as set up by owasp/modsecurity-docker
 COPY nginx/templates /etc/nginx/templates/
+COPY src/etc/modsecurity.d/modsecurity-override.conf /etc/nginx/templates/modsecurity.d/modsecurity-override.conf.template
 COPY src/etc/modsecurity.d/setup.conf /etc/nginx/templates/modsecurity.d/setup.conf.template
+COPY src/bin/healthcheck /usr/local/bin/healthcheck
+COPY nginx/docker-entrypoint.d/*.sh /docker-entrypoint.d/
 COPY src/opt/modsecurity/activate-rules.sh /docker-entrypoint.d/95-activate-rules.sh
-COPY --from=release /opt/owasp-crs /opt/owasp-crs
 
-# Alpine needs GNU 'sed' because the 'sed' program shipped with busybox does not support 'z' parameter for separating lines with a 'NUL' character.
+
 RUN set -eux; \
     apk add --no-cache \
-    sed; \
-    ln -sv /opt/owasp-crs /etc/modsecurity.d/
+        curl \
+        curl-dev \
+        libfuzzy2 \
+        libmaxminddb-dev \
+        libstdc++ \
+        libxml2-dev \
+        lmdb-dev \
+        moreutils \
+        tzdata \
+        pcre \
+        pcre2 \
+        # Alpine needs GNU 'sed' because the 'sed' program shipped with busybox does not support 'z' parameter for separating lines with a 'NUL' character.
```

Comment:
Put the comment at the top or end it with `\`, otherwise the command ends on this line.

Reply:
Hmmm, and why the build doesn't fail?

Reply:
This works as expected!